legacy/messaging/src/main/java/org/jboss/as/messaging/MigrateOperation.java

```diff
@@ -177,15 +177,14 @@ static void registerOperations(ManagementResourceRegistration registry, Resource
         registry.registerOperationHandler(new SimpleOperationDefinitionBuilder(MIGRATE, resourceDescriptionResolver)
                         .setParameters(ADD_LEGACY_ENTRIES)
                         .setReplyParameters(MIGRATION_WARNINGS_ATTR, MIGRATION_ERROR_ATTR)
-                        .setRuntimeOnly()
```

Comment:
@jmesnil Do you recall any reason for having these migrate ops runtime-only? They aren't, and correcting that doesn't break the testsuite at least. But I know there were a lot of subtle things about these ops so I don't want to break something.